### PR TITLE
hard-coded GPG key for RHEL-like systems

### PR DESCRIPTION
**Pull request by tobowers**
Checkin the GPG key to the repository until SSL is enabled on the wazuh repository.

I opened #41 a few hours ago. I think this PR mitigates the problem until the wazuh repo can have SSL on it. 

I'm not sure why the priority 1 and the disabling of protection, so maybe I missed a use-case/bug, but I wanted to get the discussion started.

### CODE REVIEW

**Comment by jlruizmlg** [comment]
Hi @tobowers first at all thanks for your contribution and you are right, we will fix this problem this week at most.

Looking your pull request i noticed that you are using the same GPG key ```RPM-GPG-KEY-OSSEC``` for all O.S versions, but in our case we have two different:

- CentOS 5/RedHat 5 ```http://ossec.wazuh.com/key/RPM-GPG-KEY-OSSEC-RHEL5``` 

- CentOS 6-7/RedHat 6-7 ```http://ossec.wazuh.com/key/RPM-GPG-KEY-OSSEC used for version 6 and 7```

**Comment by tobowers** [comment]
@jlruizmlg I updated the PR to include the rhel5 key too.

**Comment by jlruizmlg** [comment]
Thanks @tobowers! let me try and verify that all work nice.

File: manifests/repo.pp [hunk 1]
```diff
@@ -48,37 +48,56 @@
...
         yumrepo { 'ossec':
           descr    => 'WAZUH OSSEC Repository - www.wazuh.com',
           enabled  => true,
           gpgcheck => 1,
-          gpgkey   => 'http://ossec.wazuh.com/key/RPM-GPG-KEY-OSSEC-RHEL5',
+          gpgkey   => 'file:///src/ossec/RPM-GPG-KEY-OSSEC-RHEL5',
           baseurl  => 'http://ossec.wazuh.com/el/$releasever/$basearch',
-          priority => 1,
-          protect  => false,
+          require  => File["/usr/src/ossec/RPM-GPG-KEY-OSSEC"],
```
Comment (jlruizmlg) [inline manifests/repo.pp]: @tobowers we can modify this line to ```RPM-GPG-KEY-OSSEC-RHEL5``` ?

Comment (jlruizmlg) [inline manifests/repo.pp]: And also add in this block 
```
      file { "/usr/src/ossec/RPM-GPG-KEY-OSSEC-RHEL5":
        ensure  => present,
        source  => 'puppet:///modules/ossec/RPM-GPG-KEY-OSSEC-RHEL5',
        owner   => root,
        group   => root,
        mode    => 0744,
        require => File["/usr/src/ossec"]
      }
```
to keep clean the system from not necessary keys?

**Comment by jlruizmlg** [comment]
@tobowers i can merge and modify if this is ok for you.

**Comment by tobowers** [comment]
totally ok with me - I won't get to fixing this until tomorrow (I'm on european time)

**Comment by jlruizmlg** [comment]
ok thanks!

**Pull request merged**